assembl/static2/css/components/timeline.scss

```diff
@@ -38,6 +38,12 @@
 }
 
 .minimized-timeline {
+  height: 100%;
+  position: relative;
+  flex-grow: 1;
+  flex-basis: 0;
```

Comment:
Curious to see the result. Did you check the browsers compatibility? I think so...

Reply:
works on all browsers